internal/event/event.go

```diff
@@ -12,6 +12,9 @@ import (
 	"time"
 )
 
+// ErrEventProcessing is a general event processing error message.
+var ErrEventProcessing = errors.New("event could not be processed successfully, see server logs for details")
```

Comment:
This is never actually used as an `error`. For me, this is more similar to the other response messages like "POST required", thus I'd also just put it there in `listener.go`.